tests/wait_time_test.c

```diff
@@ -0,0 +1,100 @@
+#include "../headers/CuTest.h"
+
+#include "../headers/wait_time.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+
+/*Testing whether or not a person with wait-time 0 is finished*/
+void wait_test_one (CuTest *tc) {
+    passenger p0;
+    transition_system t;
+
+    t.length = 1;
+    t.passengers = (passenger*)calloc(t.length, sizeof(passenger));
+
+    p0.wait_time = 0;
+
+    t.passengers[0] = p0;
+
+    wait_time(&t);
+
+    CuAssertTrue(tc, t.passengers[0].wait_time == 0);
+    
+    free(t.passengers);
+}
+
+/*Testing if a person with a wait time above one can finsih*/
+void wait_test_two (CuTest *tc) {
+    passenger p0;
+    transition_system t;
+
+    t.length = 1;
+    t.passengers = (passenger*)calloc(t.length, sizeof(passenger));
+
+    p0.wait_time = 3;
+
+    t.passengers[0] = p0;
+
+    wait_time(&t);
+
+    CuAssertTrue(tc, t.passengers[0].wait_time == 0);
```

Comment:
Når funktionen kaldes, skal ventetiden kun formindskes med 1, hvorfor den forventede nye ventetid er 3 -1 = 2

```suggestion
    CuAssertTrue(tc, t.passengers[0].wait_time == 2);
```